src/pages/configurePatch/ConfigurePatchCore.tsx

```diff
@@ -118,6 +120,8 @@ export const ConfigurePatchCore: React.VFC<Props> = ({ patch }) => {
     });
   };
 
+  const Link = useLGButtonRouterLink(getVersionRoute(id));
```

Comment:
Would it make sense for the destination here to be the previous page? We could access it with `useNavigate()`. That would preserve the filters/pagination of the version page.

Reply:
Wow great catch y'all!